### DISCUSSION TITLE
Printing is disabled

### DISCUSSION THREAD

**Discussion by leros2004** [Alex313031/Thorium-Win#300] [General]
Hi to all, since the last 2 updates i can not print, when i click to print , in the print preview section therei is a message: print preview failed. Also all the fileds are grey out (destination. pages, layout).
I tried to roll back, the newest working version is: 124.0.6367.218

**Comment by gz83** [comment]
#266

**Reply by leros2004** [reply]
Thank you for your help. I can print again
